memcached/init.lua

```diff
@@ -109,7 +109,7 @@ local typetable = {
     readahead = {
         'number',
         function() return box.cfg.readahead end,
-        function(x) return x > 0 and x < math.pow(2, 10) end,
+        function(x) return x > 0 and x <= box.cfg.readahead end,
```

Comment:
I don't see a reason, why this buffer should be bounded by current iproto buffer size. They're independent.

However we can steal the maximum from its [validation code](https://github.com/tarantool/tarantool/blob/297c33492f8697350f78754fe3cb166f700c16ae/src/box/box.cc#L1091-L1100). It is `math.pow(2, 31) - 1`.